src/main/java/core/basesyntax/db/FruitDao.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.db;
+
+import java.util.Map;
+
+public interface FruitDao {
+    Integer putFruits(String key, Integer value);
+
+    Map<String, Integer> getFruitDB();
```

Comment:
According to the encapsulation principle, direct access to the underlying data structure should be restricted. Instead of providing a method to get the entire database, consider using methods that query the necessary information without exposing the internal storage structure. For example, you might have methods like `getQuantityByFruit` or `getAllFruits`.